source/user/mcscf.rst

```diff
@@ -188,6 +189,53 @@ See :source:`examples/mcscf/19-frozen_core.py` for a complete example.
   The `frozen` keyword of the CASSCF optimizer should not be confused with the `frozen` keyword of the FCI solver, which controls the number of orbitals that are constrained to be doubly occupied.
 
 
+Spin state of CAS wavefunction
+------------------------------
+By default, the MCSCF wavefunction uses the setting in :attr:`mol.spin` by default for the value of Sz. 
+However, there is no guarantee that a spin-adapted wave function of the desired spin multiplicity is obtained. 
+There are a few settings that can be used to tune the spin multiplicity of the wavefunction.
+
+You can change the number of alpha and beta electrons in the active space.
+The Sz value of the MCSCF wavefunction can differ from :attr:`mol.spin`.
+It's common to start from `Sz=0` RHF calculations, and then to vary the 
+number of alpha and beta electrons in the active space to solve higher 
+spin states, such as the triplet state. For example```
+
+.. code-block:: python
+
+  mol.spin = 0
+  myhf = mol.RHF().run()
+  # 5 alpha electrons, 3 beta electrons
+  mycas = mcscf.CASSCF(myhf, 6, (5, 3))
+  mycas.kernel()
+
+It may happen that a wave function of correct Sz is achieved while
+S^2 is wrong. This issue may either be caused by convergence onto
+another spin state, or spin contamination in the CAS wave function.
+
+Such issues can be circumvented with the :func:`fix_spin_` method
+of the CASCI/CASSCF class, which is able to correct the spin state
+by biasing the calculation towards the wanted state.
+
+.. code-block:: python
+
+  mol.spin = 0
+  myhf = mol.RHF().run()
+  # 5 alpha electrons, 3 beta electrons
+  mycas = mcscf.CASSCF(myhf, 6, (4, 4))
+  # Targeting triplet state
+  mycas.fix_spin_(ss=2)
+  mycas.kernel()
+
+:func:`fix_spin_` is an energy-penalty method. It may affect the efficiency and
+accuracy of the CASCI and CASSCF algorithms. It should not be used unless
```

Comment:
Just a comment: isn't it a good idea to enable it automatically if the obtained S^2 is different from the Sz*(Sz+1)?

Reply:
For the moment I tend to keep the default fci solver for mcscf the simplest one. I suspect this treatment can cause numerical instability for complicated systems. In the future it should be possible to make the mcscf object more smart to automatically set the spin treatment.